hack/test-cmd.sh

```diff
@@ -682,9 +682,7 @@ __EOF__
 
   ### Perform a rolling update with --image
   # Command
-  kubectl rolling-update frontend --image=kubernetes/pause --update-period=10ns --poll-interval=10ms "${kube_flags[@]}"
-  # Post-condition: current image IS kubernetes/pause
-  kube::test::get_object_assert 'rc frontend' '{{range \$c:=$rc_container_image_field}} {{\$c.image}} {{end}}' ' +kubernetes/pause +'
+  [ ! kubectl rolling-update frontend --image=kubernetes/pause --timeout=1s --update-period=10ns --poll-interval=10ms "${kube_flags[@]}" || true
```

Comment:
Sorry I cant see where this starting square bracket is closed.
Also, will be good to add a comment here saying that this is expected to fail since the pods will never become ready